### DISCUSSION TITLE
How to remap jump word to alt+right(and the same for left)

### DISCUSSION THREAD

**Discussion by tipa40** [gokcehan/lf#1702] [Q&A] [answered]
Hi
I tried:
map \<a-right\> cmd-word
and ”all sorts of combinations” but they all fail. :(

ctrl instead of alt seems to work equally bad for me.

**Comment by DusanLesan** [comment]
I have just tried `map <a-right> $notify-send "RIGHT!"` and it works ok for me. Not sure if your terminal could cause the problem. [Key mappings section](https://github.com/gokcehan/lf/blob/master/doc.md#key-mappings) in documentation mentions the OS intercepting combinations and that sounds like a most likely cause to me

**Comment by tipa40** [comment]
Thanks for your quick response! I also tried `map <a-right> $notify-send "RIGHT!"` and I can see that it works for me too.
Could it be that I should give anything else instead of `cmd-word` or do I have to call it differently, or something else preventing it?

I have tried it using foot and alacritty with the same nothing-happens result, in SwayWM.

My test scenario is trying to navigate words in the command line mode and for instance in `:rename` mode where alt+f/b is working well.

**Comment by DusanLesan** [comment, marked as answer]
> Could it be that I should give anything else instead of `cmd-word` or do I have to call it differently, or something else preventing it?

That is right. I have overlooked that you wanted to edit a map for a command line command. 

Try with `cmap`: 
```
cmap <a-right> cmd-word
cmap <a-left> cmd-word-back
```

**Comment by tipa40** [comment]
Wow, now it works flawlessly! I am super happy! Thank you very much for your help! 😊👍🏻☀️
